frontend/src/model/DraftNewReleaseDialog.tsx

```diff
@@ -13,35 +13,56 @@ import {
   TextField,
   Typography,
 } from '@mui/material'
-import { ChangeEvent, FormEvent, useState } from 'react'
+import { FormEvent, useState } from 'react'
 import semver from 'semver'
 
-import { Artefact } from '../../types/types'
-import FileInput from '../common/FileInput'
+import { postRelease } from '../../actions/release'
+import useNotification from '../../src/common/Snackbar'
+import { ReleaseInterface } from '../../types/types'
+import MultiFileInput from '../common/MultiFileInput'
 
-export default function DraftNewReleaseDialog({ open, handleClose }: { open: boolean; handleClose: () => void }) {
+export default function DraftNewReleaseDialog({
+  open,
+  handleClose,
+  modelId,
+}: {
+  open: boolean
+  handleClose: () => void
+  modelId: string
+}) {
   const [releaseName, setReleaseName] = useState('')
   const [semanticVersion, setSemanticVersion] = useState('')
   const [releaseNotes, setReleaseNotes] = useState('')
   const [isMinorRelease, setIsMinorRelease] = useState(false)
-  const [artefact, setArtefact] = useState<Artefact | undefined>()
+  const [artefacts, setArtefacts] = useState<File[]>()
 
-  function onSubmit(event: FormEvent<HTMLFormElement>) {
+  const sendNotification = useNotification()
```

Comment:
I believe we're trying to move away from snackbars, see my other comment about this for more info.